examples/auth/src/App.tsx

```diff
@@ -136,7 +136,7 @@ function RequireAuth({ children }: { children: JSX.Element }) {
     // trying to go to when they were redirected. This allows us to send them
     // along to that page after they login, which is a nicer user experience
     // than dropping them off on the home page.
-    return <Navigate to="/login" state={{ from: location }} />;
+    return <Navigate to="/login" state={{ from: location }} replace={true} />;
```

Comment:
```suggestion
    return <Navigate to="/login" state={{ from: location }} replace />;
```

The `={true}` is superfluous in JSX, so you can just drop it.